src/engine/Input/Pointer.ts

```diff
@@ -379,40 +385,22 @@ export class Pointers extends Class {
    private _handleWheelEvent(eventName: string, eventArr: WheelEvent[]) {
       return (e: MouseWheelEvent) => {
          // Should we prevent page scroll because of this event
-         if (this._engine.pageScrollPreventionMode === PageScrollPreventionMode.Page ||
-             (this._engine.pageScrollPreventionMode === PageScrollPreventionMode.Canvas && e.target === this._engine.canvas)) {
+         if (this._engine.pageScrollPreventionMode === ScrollPreventionMode.All ||
+             (this._engine.pageScrollPreventionMode === ScrollPreventionMode.Canvas && e.target === this._engine.canvas)) {
             e.preventDefault();
          }
 
          var x: number = e.pageX - Util.getPosition(this._engine.canvas).x;
          var y: number = e.pageY - Util.getPosition(this._engine.canvas).y;
          var transformedPoint = this._engine.screenToWorldCoordinates(new Vector(x, y));
 
-         var deltaX = 0;
-         var deltaY = 0;
-         var deltaZ = 0;
+         // Support
```

Comment:
Can we add some comments that explain what the normalization factor is for? For posterity.